hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/ContextTransactionProcessor.java

```diff
@@ -132,10 +130,6 @@ public CallOutcome call() {
                 result = result.withSignerNonce(sender.getNonce());
             }
 
```

Comment:
I must be missing it but where do we refund the fees if the `EthereumTransaction` is successful?